MidCamp/Hatter
# create sub event list component

Sub event list component

### Testing instructions
* `npm run butler`
* Navigate to the styleguide
* Observe you see a link for Sub Events List under components
* Click Sub Events List
* Observe you see a component for a Sub Event List

_This will require sub events to have images for them. We might want to have a backup pattern or two that we can use if we don't have images for an event._
